Running tests with go workspace results in errors

Steps to reproduce:
1. Create a new go "workspace" e.g.: `~/openwhisk` <-- this is the GOPATH
1. Clone this repo into `$GOPATH/src/github.com/apache/incubator-openwhisk-cli`
1. `cd  $GOPATH/src/github.com/apache/incubator-openwhisk-cli`
1. `make build`
1. `make test`
1. `make native_test`
```Launch the native tests for the commands.
go test -v ./... -tags=native
?       github.com/apache/incubator-openwhisk-cli       [no test files]
?       github.com/apache/incubator-openwhisk-cli/commands      [no test files]
=== RUN   TestWskExist
--- PASS: TestWskExist (0.00s)
=== RUN   TestHelpUsageInfoCommand
--- FAIL: TestHelpUsageInfoCommand (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xa0 pc=0x6b6b56]

goroutine 51 [running]:
testing.tRunner.func1(0xc4201821e0)
        /home/mcweeney/.asdf/installs/golang/1.9.2/go/src/testing/testing.go:711 +0x2d2
panic(0x7a0320, 0xa11740)
        /home/mcweeney/.asdf/installs/golang/1.9.2/go/src/runtime/panic.go:491 +0x283
github.com/apache/incubator-openwhisk-cli/vendor/github.com/stretchr/testify/assert.isNumericType(0x0, 0x0, 0x9e1901)
        /home/mcweeney/github/mcdan/openwhisk/src/github.com/apache/incubator-openwhisk-cli/vendor/github.com/stretchr/testify/assert/assertions.go:310 +0x26
github.com/apache/incubator-openwhisk-cli/vendor/github.com/stretchr/testify/assert.formatUnequalValues(0x0, 0x0, 0x7993c0, 0xc4200f18e0, 0x0, 0x0, 0xc42017e2a0, 0xc420051e80)
        /home/mcweeney/github/mcdan/openwhisk/src/github.com/apache/incubator-openwhisk-cli/vendor/github.com/stretchr/testify/assert/assertions.go:300 +0xbb
github.com/apache/incubator-openwhisk-cli/vendor/github.com/stretchr/testify/assert.Equal(0x9e1d20, 0xc4201821e0, 0x0, 0x0, 0x7993c0, 0xc4200f18e0, 0xc420063f08, 0x1, 0x1, 0x8f7498)
        /home/mcweeney/github/mcdan/openwhisk/src/github.com/apache/incubator-openwhisk-cli/vendor/github.com/stretchr/testify/assert/assertions.go:281 +0x10c
github.com/apache/incubator-openwhisk-cli/tests/src/integration.TestHelpUsageInfoCommand(0xc4201821e0)
        /home/mcweeney/github/mcdan/openwhisk/src/github.com/apache/incubator-openwhisk-cli/tests/src/integration/command_test.go:50 +0x135
testing.tRunner(0xc4201821e0, 0x81bfc0)
        /home/mcweeney/.asdf/installs/golang/1.9.2/go/src/testing/testing.go:746 +0xd0
created by testing.(*T).Run
        /home/mcweeney/.asdf/installs/golang/1.9.2/go/src/testing/testing.go:789 +0x2de
FAIL    github.com/apache/incubator-openwhisk-cli/tests/src/integration 0.009s
?       github.com/apache/incubator-openwhisk-cli/tests/src/integration/common  [no test files]
?       github.com/apache/incubator-openwhisk-cli/wski18n       [no test files]
Makefile:29: recipe for target 'native_test' failed
```

This is actually just due to the fact that while it checks for the existence of the wsk cli at `wsk.Dir + "/" + wsk.Path` it calls that same cli with: `exec.Command(wsk.Path`

--- a/tests/src/integration/common/wsk.go
+++ b/tests/src/integration/common/wsk.go
@@ -60,7 +60,7 @@ func (wsk *Wsk)Exists() bool {
 func (wsk *Wsk)RunCommand(s ...string) ([]byte, error) {
     cs := wsk.Arg
     cs = append(cs, s...)
-    command := exec.Command(wsk.Path, cs...)
+    command := exec.Command(wsk.Dir + "/" + wsk.Path, cs...)
     command.Dir = wsk.Dir
     return command.CombinedOutput()
 }

